### PR TITLE
Mhs/cumulus 2224/add extended metadata type to reconciliation report

### PR DESCRIPTION
**Pull request by flamingbear**
**Summary:** Adds "EXTENDED METADATA" type to granule files' comparisons

Addresses [CUMULUS-2224: Not all files are being found in CMR when they exist in cumulus during reconciliation reporting](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2224)

## Changes

* Adds test and code to include the relatedUrls Type 'EXTENDED METADATA' 

## PR Checklist

- [ x] Update CHANGELOG
- [ x] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: packages/api/tests/lambdas/test-create-reconciliation-report.js
Comment (jennyhliu) [inline packages/api/tests/lambdas/test-create-reconciliation-report.js:1442]: The variable is `matchingFilesInCmr`, need another variable, maybe `ignoredFilesInCmr`?

Comment (flamingbear) [reply packages/api/tests/lambdas/test-create-reconciliation-report.js]: good point. ed2dd430a

**Review by jennyhliu — APPROVED** [review]
LGTM

**Comment by flamingbear** [comment]
waiting for @laurenfrederick approval to merge. (we might not want to update this)

**Pull request merged**
